stock_storage_type/models/stock_quant.py

```diff
@@ -49,8 +49,15 @@ def _check_storage_capacities(self):
             lots_in_location = other_quants_in_location.mapped("lot_id")
             capacity_fails = []
             for capacity in allowed_capacities:
+                allow_new_product = capacity.storage_category_id.get_allow_new_product(
```

Comment:
you can remove the for loop
```suggestion
            allow_new_product = location.computed_storage_category_id.get_allow_new_product(
```

Reply:
Done